app/helpers/pipeline_runs_helper.rb

```diff
@@ -129,7 +134,7 @@ def upload_dag_json_and_return_job_command(dag_json, dag_s3, dag_name, key_s3_pa
 
   def install_pipeline(commit_or_branch)
     "pip install --upgrade git+git://github.com/chanzuckerberg/s3mi.git; " \
-    "cd /mnt; " \
+    "cd /mnt; rm -rf *; df -h; " \
```

Comment:
I guess this probably is not an issue, but `rm -rf *` won't remove hidden files at root level, correct?

Reply:
Correct. But we don't care because the only files of significant size would not be hidden.